README.md

```diff
@@ -38,4 +38,27 @@ And in another terminal run the client:
 $ npm run dev
 ```
 
-Now visit the page `http://localhost:8080/`
\ No newline at end of file
+Now visit the page `http://localhost:8080/`
+
+# How to run deploy through bothub
+
+Bothub, a chatbot hosting service for developers
+
+Create a [bothub](https://bothub.studio/) account
+
+First install bothub-cli
+```
+$ pip install bothub-cli
+```
+Then you'll need to assign these values
+```
+$ bothub channel add line --channel-id=<channel id> \
+                        --channel-secret=<channel secret> \
+                       --channel-access-token=<channel access token>
+```
+Inside the MyBot directory run this command in powershell
+```
+$ bothub deploy
+```
+Your bot should be running now
+![QR Code](./imgs/linebot-QR.png)
```

Comment:
You are missing the `.bothub` configure file that needs to be present when you deploy which get's created when we configure bothub. We don't need to run the config, just explain what credential files need to be stored.